### PR TITLE
[docs] Add changelog for v0.14.0

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,179 @@
...
+* The blur event is now raised on time when an input is hidden in IE ([#1275](https://github.com/DevExpress/testcafe/issues/1275))
+* TestCafe no longer fails if a client function argument contains ES6 method syntax ([#1279](https://github.com/DevExpress/testcafe/issues/1279))
+* Errors that occur during browser provider initialization are now caught ([#1282](https://github.com/DevExpress/testcafe/issues/1282))
+* Click on the debugger panel no longer affects the tested page ([#1200](https://github.com/DevExpress/testcafe/issues/1200))
+* An unhandled error no longer occurs when running a fixture without tests ([#1302](https://github.com/DevExpress/testcafe/issues/1302))
+* The `input` event is now raised when the value of a `select` element is changed ([#1311](https://github.com/DevExpress/testcafe/issues/1311))
+* You can now perform actions with ShadowDOM elements ([#1312](https://github.com/DevExpress/testcafe/issues/1312))
+* Server no longer responses with status 222 when window.fetch() is called in Chrome ([#1134](https://github.com/DevExpress/testcafe/issues/1134))
```
Comment (inikulin) [inline CHANGELOG.md]: `no longer responds`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,179 @@
...
+});
+```
+
+### Bug Fixes
+
+* Fixed a bug that led to an incorrect callstack in test run report ([#1226](https://github.com/DevExpress/testcafe/issues/1226))
+* Cursor is now hidden on screenshots created using the `t.takeScreenshot` action ([#1245](https://github.com/DevExpress/testcafe/issues/1245))
+* Error no longer appears when selecting a non-existent child by index ([#1240](https://github.com/DevExpress/testcafe/issues/1240))
+* The blur event is now raised on time when an input is hidden in IE ([#1275](https://github.com/DevExpress/testcafe/issues/1275))
+* TestCafe no longer fails if a client function argument contains ES6 method syntax ([#1279](https://github.com/DevExpress/testcafe/issues/1279))
```
Comment (inikulin) [inline CHANGELOG.md]: `ES6 method syntax` -> `ES6 class method syntax

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,179 @@
...
+* Errors that occur during browser provider initialization are now caught ([#1282](https://github.com/DevExpress/testcafe/issues/1282))
```
Comment (inikulin) [inline CHANGELOG.md]: `TestCafe now reports errors that occur during browser provider initialization`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,179 @@
...
+    await t
+        .debug()
+        .click('#populate')
+        .click('#submit-button');
+});
+```
+
+In the footer, you'll find buttons that allow you to continue test execution or step to the next test action.
+
+![Page Footer in the Debug Mode](media/client-debugging-footer.png)
```
Comment (inikulin) [inline CHANGELOG.md]: I guess we need for screenshot with new buttons to be landed? Or it can be updated independently?

Comment (VasilyStrelyaev) [inline CHANGELOG.md]: I'll update the screenshot separately

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,179 @@
...
+
+test('My test', async t => {
+    await t
+        .debug()
+        .click('#populate')
+        .click('#submit-button');
+});
+```
+
+In the footer, you'll find buttons that allow you to continue test execution or step to the next test action.
```
Comment (inikulin) [inline CHANGELOG.md]: We need a screenshot and mention of current breakpoint shown in console

**Comment by VasilyStrelyaev** [comment]
ping @churkin @kirovboris

**Pull request merged**